WordPress/src/main/res/layout/scan_list_scan_state_item.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <ImageView
+        android:id="@+id/scan_state_icon"
+        android:layout_width="@dimen/scan_state_icon_size"
+        android:layout_height="@dimen/scan_state_icon_size"
+        android:layout_marginTop="@dimen/margin_extra_medium_large"
+        android:contentDescription="@string/scan_state_icon"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <com.google.android.material.textview.MaterialTextView
+        android:id="@+id/scan_state_title"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/margin_extra_large"
+        android:textStyle="bold"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/scan_state_icon"
+        tools:text="Don't worry about a thing" />
```

Comment:
Question (❓ ): I am not sure what is the best or agreed approach here, but what if the actual string res was used for both `tools:text` attributes instead of a hard-coded string that can end up being out-dated?

```
tools:text="@string/scan_idle_no_threats_found_title"
...
tools:text="@string/scan_idle_no_threats_found_description"
```

Reply:
In f680209